components/jupyter-web-app/releaser/.gitignore

```diff
@@ -0,0 +1,4 @@
+/lib
```

Comment:
Why are you ignoring /lib?

Reply:
I saw in one of ksonnet's [PRs](https://github.com/ksonnet/ksonnet/pull/406) that this folder is can be automatically generated. This gitignore was created from ksonnet. Should I not ignore it?